src/FacadeVFS.js

```diff
@@ -3,6 +3,9 @@ import * as VFS from './VFS.js';
 
 const AsyncFunction = Object.getPrototypeOf(async function(){}).constructor;
 
+// Unix epoch in milliseconds as a BigInt.
+const UNIX_EPOCH = 24405875n * 8640000n;
```

Comment:
It would be good to add a comment on where this comes from.

The best source I've found is here:
https://github.com/sqlite/sqlite/blob/e57527c14f7b7cfa6e32eeab5c549d50c4fa3674/src/os_unix.c#L6872-L6882

Basically, we need milliseconds since Julian epoch, which is 2440587.5 days before the unix epoch.

Reply:
Made the description slightly better and added the reference.